src/google/cloud/ndb/_datastore_api.py

```diff
@@ -654,8 +654,15 @@ def commit(self, retries=None, timeout=None):
             timeout (float): Timeout, in seconds, to pass to gRPC call. If
                 :data:`None` is passed, will use :data:`_DEFAULT_TIMEOUT`.
         """
-        if not self.mutations:
-            return
+        # It's tempting to do something like:
```

Comment:
Found this bug in the system tests for this feature. Left in this warning for future optimizers.

Reply:
Thanks!